src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/XML/ParseTreeDescription.vb

```diff
@@ -613,6 +613,10 @@ Public Class ParseNodeChild
             End If
             Return _childKind
         End Get
+
+        Set
```

Comment:
>Set [](start = 8, length = 3)

Could we avoid making this mutable? It seems all the other properties on this type are readonly.

Reply:
i'm not sure a good way how to do that :(

it's admittedly not pleasant what i'm doing.  i'm effecting going in and munging hte child list in-place.  However, this is only for a tree which is only used for grammar generation.  let me see what i can do.

Reply:
ok, i have an idea :)

Reply:
Done!